tsconfig-strict.json

```diff
@@ -264,7 +264,6 @@
     "core/templates/domain/story/StoryObjectFactorySpec.ts",
     "core/templates/domain/story/story-domain.constants.ts",
     "core/templates/domain/story/story-node.model.ts",
-    "core/templates/domain/story/story-node.model.spec.ts",
```

Comment:
Do not remove the file from here, rather fix the TS errors in that file.

Reply:
Added this file back now. Actually, I removed it in order to test the following block of code:
```
  _checkValidNodeId(nodeId: string): boolean {
    if (typeof nodeId !== 'string') {
      return false;
    }
```